README.md

```diff
@@ -1,98 +1,187 @@
- -  Chatting with us real-time.
-    Please wait around for a response to your question as many folks
-    on the channel are doing other things and it may take a while for
-    them to notice your question.
-    The following paths all take you to the same chat room:
-     - Using an IRC client and joining the [#qgis](http://webchat.freenode.net/?channels=#qgis) channel on irc.freenode.net.
-     - Using a Matrix client and joining the [#qgis:matrix.org](http://matrix.to/#/#qgis:matrix.org) room.
-     - Using [Gitter](https://gitter.im/qgis/QGIS?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) chat.
+### Bug reporting and bug fixing
 
-## Contribute
+You can help us **by submitting bug reports or fixing bugs** in the [QGIS bug tracker](https://github.com/qgis/QGIS/issues/).
 
-QGIS is on GitHub at https://github.com/qgis/QGIS. If you wish to contribute
-patches you can [fork the project](https://help.github.com/forking/), make your changes, commit to your
-repository, and then [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/). The development team can then review your contribution and commit it upstream as appropriate.
+### New features and enhancements
+
+If you wish to contribute patches you can [fork the project](https://help.github.com/forking/), make your changes, commit to your repository, and then [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/). The development team can then review your contribution and commit it upstream as appropriate.
 
 If you commit a new feature, add [FEATURE] to your commit message AND give a clear description of the new feature. A webhook will automatically create an issue on the QGIS-Documentation repo to tell people to write documentation about it.
 
-If you are not a developer, there are many other possibilities which do not require programming skills to help QGIS to evolve. Check our [project homepage for more information](http://qgis.org/en/site/getinvolved/index.html).
+For large-scale changes, you can open a [QEP (QGIS Enhancement Proposal)](https://github.com/qgis/QGIS-Enhancement-Proposals). QEPs are used in the process of creating and discussing new enhancements or policy for QGIS. 
 
-### Building from source
+### Translations
 
-The [building guide](INSTALL.md) can be used to get started with building QGIS from source.
+Please help translate QGIS to your language. At this moment about forty languages are already available in the Desktop user interface and about eighty languages are available in transifex ready to be translated.
+
+The [translation](https://qgis.org/en/site/getinvolved/translate.html) process is managed by the [Translation Team](https://qgis.org/en/site/getinvolved/governance/governance.html#gui-translation) and all the activities are done under the [Transifex](https://www.transifex.com/) platform.
+
```

Comment:
Is it worth to mention that every language that is translated at 35% is automatically pulled in the release? People often wonder why their language is not pulled or not pulled anymore.

Reply:
Interesting. Feels to me like it's going a bit too much into the weeds to put this into the overall README.md, but seems like a good point for the "how to contribute effectively if you're not a developer" landing page to be created (somewhere?) as per comment elsewhere in this discussion.